.github/workflows/terraform-plan-env.yml

```diff
@@ -31,6 +60,10 @@ jobs:
         # The post includes a link to the workflow run where the plan was
         # generated, and the same comment will be updated each time a new commit
         # on the PR branch triggers a re-plan.
+      - name: show versions
+        run: |
+          echo "TF_VAR_postgrest=$TF_VAR_postgrest_image"
+          echo "TF_VAR_clamav=$TF_VAR_clamav_image"
```

Comment:
```suggestion
          echo "TF_VAR_postgrest_image=$TF_VAR_postgrest_image"
          echo "TF_VAR_clamav_image=$TF_VAR_clamav_image"
```

Reply:
This one's a bug... Terraform won't like supplying a variable value for a variable name that's not declared, and omitting a value for a variable where a value is required.